BloodyMary/Utility/PDFViewController.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  BloodyMary
+//
+//  Copyright © TheInkedEngineer. All rights reserved.
+// 
+
+import UIKit
+import PDFKit
+
+/// A `PDFViewController` is `UIViewController` with an integrated `PDFView` inside of it.
+/// It is assumed that this ViewController will always be inside a navigaiton controller and therefore the `dismissButtonItem` is set as the `leftBarButtonItem` of the `navigationItem`.
+/// THe `dismissButtonItem` is customisable.
+public class PDFViewController: UIViewController {
+  
+  // MARK: - Properties
+  
+  /// The back button to dismiss the `PDFView`.
+  public var dismissButtonItem = UIBarButtonItem(title: "Dismiss", style: .plain, target: nil, action: #selector(tappedDismissButton)) {
```

Comment:
Instead of this, maybe it's better to use:

```
init(barButtonSystemItem: UIBarButtonItem.SystemItem, target: Any?, action: Selector?)
```

with `barButtonSystemItem`:
- iOS < 13: .done
- iOS >= 13: .close